app/src/main/java/mozilla/lockbox/store/DefaultPreferencesStore.kt

```diff
@@ -0,0 +1,42 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ *  License, v. 2.0. If a copy of the MPL was not distributed with this
+ *  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package mozilla.lockbox.store
+
+import android.content.Context
+import android.content.SharedPreferences
+import android.preference.PreferenceManager
+import io.reactivex.Observable
+import io.reactivex.disposables.CompositeDisposable
+import mozilla.lockbox.flux.Dispatcher
+
+abstract class DefaultPreferencesStore(
```

Comment:
😬 ooh, this is getting implemented both here and in #157 right now. don't know the best way to reconcile.